File: talleres c++/ciclofor.cpp
#include <iostream>
#include <cmath> // Para usar pow() para la potenciación
using namespace std;

int main() {
    int A, C;
    
    // Solicitar el primer valor
    cout << "Por favor ingrese el primer valor: ";
    cin >> A;

    // Solicitar el segundo valor
    cout << "Por favor ingrese el segundo valor: ";
    cin >> C;

    // Calcular la potencia
    int valor = pow(A, C);  // Usamos la función pow() de la librería <cmath>
    
    // Mostrar el resultado
    cout << "La potencia de " << A << " sobre " << C << " es: " << valor << endl;

    return 0;
}

File: talleres java/ciclofor.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        
        // Solicitar el primer valor
        System.out.print("Por favor ingrese el primer valor: ");
        int A = scanner.nextInt();
        
        // Solicitar el segundo valor
        System.out.print("Por favor ingrese el segundo valor: ");
        int C = scanner.nextInt();
        
        // Calcular la potencia
        int resultado = (int) Math.pow(A, C);  // Math.pow retorna un double, lo convertimos a int
        
        // Mostrar el resultado
        System.out.println("La potencia de " + A + " sobre " + C + " es: " + resultado);
        
        scanner.close();  // Cerrar el scanner
    }
}